cli/tests/testdata/run/092_import_map_unmapped_bare_specifier.ts.out

```diff
@@ -1,4 +1,4 @@
-error: Uncaught (in promise) TypeError: Relative import path "unmapped" not prefixed with / or ./ or ../ and not in import map from "file://[WILDCARD]/092_import_map_unmapped_bare_specifier.ts"
```

Comment:
The last part of this error message was lost because of [this commit](https://github.com/denoland/deno/pull/20728/commits/91aaf7356cb9741718cad6cdb787bdb3e348cd94) (We now ignore `ImportMapError::UnmappedBareSpecifier` because that can't be handled by bare node specifier check implemented in `deno_graph`.

I drafted [a PR](https://github.com/denoland/deno_graph/pull/298) to handle `ImportMapError::UnmappedBareSpecifier` in `deno_graph`, but I'm not sure it's worth the complexity